wpimath/src/test/native/cpp/controller/ElevatorFeedforwardTest.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include <cmath>
+
+#include <gtest/gtest.h>
+
+#include "frc/controller/ElevatorFeedforward.h"
+#include "units/acceleration.h"
+#include "units/length.h"
+#include "units/time.h"
+
+namespace frc {
+
+TEST(ElevatorFeedforwardTest, Calculate) {
+  auto Ks = 0.5_V;
```

Comment:
If you declare these as `static constexpr auto`, you may be able to move them above the test definitions to deduplicate them in the same way as the Java tests.